src/user-ratings/UserRatings.js

```diff
@@ -93,3 +95,10 @@ export default class UserRatings extends Component {
       }
   }
 }
+
+UserRatings.propTypes = {
+  fetchUserRatings: PropTypes.func,
+  movieID: PropTypes.string,
+  userStatus: PropTypes.object,
+  movieRatings: PropTypes.arrayOf(PropTypes.object),
+}
```

Comment:
Nice incorporation of PropTypes here. I think the arrayOf on 103 will help make sure we have the right type of array. Could help with any future bugs